src/test/java/com/google/api/generator/gapic/dummy/BUILD.bazel

```diff
@@ -1,3 +1,5 @@
+load("//:rules_bazel/java/java_diff_test.bzl", "update_golden")
```

Comment:
That makes sense to me, this `junit_runner` will be called for updating every goldens files, I moved it to `root/BUILD.bazel` so that `rules_bazel/java/java_diff_bzl` can directly use it (hard-code the runner) instead of declaring this binary for each package.